templates/base.html

```diff
@@ -4,6 +4,26 @@
 <html lang="en">
     <head>
         <title>{% block title %}PERDIEM - INVEST IN MUSIC{% endblock %}</title>
+        <meta name="description" content="PerDiem is a platform where you can invest in music. Earn money by discovering and supporting your favorite artists." />
+
+        <!-- Google+ data -->
+        <meta itemprop="name" content="PERDIEM - INVEST IN MUSIC">
+        <meta itemprop="description" content="PerDiem is a platform where you can invest in music. Earn money by discovering and supporting your favorite artists.">
+        <meta itemprop="image" content="http://www.investperdiem.com/img/social-icon.jpg">
```

Comment:
Nope, it should be fine. In prod, the HTML that crawlers will see will be the same as this. It's just that in other environments the HTML will be different.